docs/pages/concept.rst

```diff
@@ -215,7 +215,9 @@ Short, easy, and readable:
   that all non-empty lists with ``int`` elements
   will be treated as ``ListOfInt``
 - In runtime ``ListOfInt`` does not exist, because it is phantom!
-  In reality it is just ``List[int]``
+  In reality it is just ``Sequence[int]``.
+  Notice that newer versions of ``phantom-types`` do not accept
+  the mutable ``List`` type constructor for this purpose.
```

Comment:
Let's also add:

`Because you can add items of other types to the list after the validation is done, so it is not safe`.

Reply:
Done.